napari/components/_layers_list/view/list.py

```diff
@@ -123,15 +126,14 @@ def dropEvent(self, event):
 
 
 class QtDivider(QFrame):
```

Comment:
Why don't you move this to the `stylesheet.qss` then you wouldn't need to import the palettes here. I think we just missed this one when we were making the `stylesheet.qss`